config.toml.in

```diff
@@ -7,6 +7,8 @@
 # The storage section controls the configuration of the backend.
 [storage]
 type = "file"
+auto_backup = true
```

Comment:
Hmm, we have a `auto_backup` and `max_backups`.
Now imagine we decide to add a new `backup_format` to configure the backup-date format (or yet another backup directive). In my opinion it would make sense to group them in a more specific section like `[general.backup]`.
What do you think?

```toml
[general]
auto_backup = true
max_backups = 100
```

vs 

```toml
[general.backup]
auto = true
max = 100
```